README.md

```diff
@@ -92,6 +92,7 @@ will be returned. If you target one element, that instance will be returned.
     position: 'auto',
     resetScrollPosition: true,
     regexFilter: null,
+    callbackFilterAddItem: null,
```

Comment:
Thinking about it - I think this should just be named `addItemFilter` to align with `regexFilter` 🤔 What do you think?

Reply:
@jshjohnson - Agree. I'll update